src/ui_components/search_input.js

```diff
@@ -1,6 +1,14 @@
 import Suggest from "../adapters/suggest";
 import PanelManager from "../proxies/panel_manager";
-import BragiPoi from "../adapters/poi/bragi_poi";
+
+const MAPBOX_RESERVED_KEYS = [
+    37 // ←
+  , 38 // ↑
+  , 39 // →
+  , 40 // ↓
+  , 107 // -
+  , 109 // +
+]
```

Comment:
we should also add `=` (https://docs.mapbox.com/mapbox-gl-js/api/#keyboardhandler)

Reply:
i have miss this one ;(